velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -257,6 +257,23 @@ class PlanBuilder {
       const std::string& outputDirectoryPath,
       dwio::common::FileFormat fileFormat = dwio::common::FileFormat::DWRF);
 
+  /// Adds a TableWriteNode to write certain input columns into an
+  /// un-partitioned un-bucketed Hive table. Specify the column and aggregation
+  /// function to collect column statistics.
+  ///
+  /// @param inputColumns Input columns to write.
+  /// @param outputDirectoryPath Path to a directory to write data to.
+  /// @param fileFormat File format to use for the written data.
+  /// @param statsAggField Column to collect statistics during write.
```

Comment:
It is more convenient to specify aggregations using SQL strings: min(c0), max(c0), etc. See other similar APIs.